### PR TITLE
Refine doc and fix bug for testing MedianstopAssessor

### PR DESCRIPTION
**Pull request by Sundrops**

### CODE REVIEW

File: src/sdk/pynni/nni/medianstop_assessor/test.py [hunk 1]
```diff
@@ -31,11 +31,11 @@ def test():
...
     #       [1,1,1,1,1,1,1,1,1,1]]
 
-    assessor = MedianstopAssessor(FLAGS.start_step, FLAGS.optimize_mode)
-    for i in range(4):
+    assessor = MedianstopAssessor(FLAGS.optimize_mode, FLAGS.start_step)
+    for i in range(len(lcs)):
         #lc = []
         to_complete = True
-        for k in range(10):
+        for k in range(FLAGS.start_step):
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/medianstop_assessor/test.py]: `FLAGS.start_step` -> `len(lcs[0])`

Comment (QuanluZhang) [reply src/sdk/pynni/nni/medianstop_assessor/test.py]: `FLAGS.start_step` is internal used by `MedianstopAssessor`

Comment (Sundrops) [reply src/sdk/pynni/nni/medianstop_assessor/test.py]: @QuanluZhang  Fixed

File: docs/zh_CN/Assessor/MedianstopAssessor.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 ## Median Stop
 
-Medianstop 是一种简单的提前终止 Trial 的策略，可参考[论文](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46180.pdf)。 如果 Trial X 的在步骤 S 的最好目标值比所有已完成 Trial 的步骤 S 的中位数值明显要低，这个 Trial 就会被提前停止。
\ No newline at end of file
+Medianstop 是一种简单的提前终止 Trial 的策略，可参考[论文](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46180.pdf)。 如果 Trial X 在步骤 S 的最好目标值低于所有已完成 Trial 前S个步骤目标平均值的中位数，这个 Trial 就会被提前停止。
```
Comment (QuanluZhang) [inline docs/zh_CN/Assessor/MedianstopAssessor.md]: thanks for your fix, but could you revert this change, because we have a dedicated pipeline for the update of chinese doc

Comment (QuanluZhang) [reply docs/zh_CN/Assessor/MedianstopAssessor.md]: we will update this part later :)

Comment (Sundrops) [reply docs/zh_CN/Assessor/MedianstopAssessor.md]: @QuanluZhang Done.
